components/Sections/RichText/RichTextStyle.module.css

```diff
@@ -0,0 +1,3 @@
+.richtext {
+  font: bold;
```

Comment:
Don't commit this if it is only for testing (we don't want all RichTexts to be bold).